package.json

```diff
@@ -1,5 +1,5 @@
 {
-  "name": "dcf",
+  "name": "dcn-dcf",
```

Comment:
Should it be `digitalcampusframework-dcf` now with the repo rename?

Reply:
@macburgee1 are we gonna rename or kill this PR?

Reply:
I defer